core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1247,6 +1247,20 @@ private static RelDataType deriveTypeMapFromArrays(SqlOperatorBinding opBinding)
           OperandTypes.STRING,
           SqlFunctionCategory.STRING);
 
+  @LibraryOperator(libraries = {BIG_QUERY})
```

Comment:
How about adding some java doc for those two functions? It helps others to understand better without browsing the JIRA description.

Reply:
@ILuffZhe Thanks for your suggestion, I have updated my branch. Please take a look when you have time.